portfolio/src/main/webapp/index.html

```diff
@@ -7,12 +7,55 @@
     <script src="script.js"></script>
   </head>
   <body>
-    <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
+    <div id="title">
+      <h1>Welcome to Jason's Portfolio</h1>
     </div>
+
+    <div id="header">
+      <img class="circular--square" src="../Images/headshot.png" alt="My headshot">
+      <p> Jason Ah Chuen <br>
+          Stanford University | Class of 2022 <br>
+          B.S. Candidate | Computer Science <br>
+          jahchuen (at) stanford (dot) edu
+      </p>
+    </div>
+
+    <div id="content1">
+      <h3> I am a sophomore student at Stanford University, passionate about CS + social good, 
+          entrepreneurship and so much more. </h3>
+    </div>
+
+    <div id="experiences">
+      <h1> Experience </h1>
+      <h3> Stanford Artificial Intelligence Laboratory </h3>
+      <h5> <i> Robotics/AI intern </i> </h5>
+      <p> Developed haptic perception methods on a robot to detect and learn surface features like geometry, texture and
+friction. Implemented software on Dynamixel motors, designed experiments for the robot, collected data sets, and
+found new approaches to distinguish between those overlapping features using Fast Fourier Transforms and classify
+each feature using statistical methods and machine learning. Currently working on publishing a paper. </p>
+      <img src="../Images/poster.png" alt="My research poster" class="center" width="500px" "height=170px">
+    </div>
+
+    <div id="passions">
+      <h1> Passions </h1>
+      <h3> Tennis </h3>
+      <p> I've been playing tennis since I was six. I got to travel a lot around the Indian Ocean and the African continent to represent my country's national team in tennis. </p>
+      <img src="../Images/tennis.jpg" alt="Me playing tennis" class="center" width="500px" "height=500px">
+      <h3> Djason alouda </h3>
+      <p> I've started DJ'ing last Summer. Now I DJ regularly for late night entertainment at Stanford dining halls, and DJ occasionally at parties.
+          Back home, I've also performed in night clubs. My favorite music taste is Future House and Progressive House music.
+      <img src="../Images/djason_alouda.jpeg" alt="Me DJ'ing" class="center" width="300px" "height=500px">
+    </div>
+
+    <div id="content1">
+      <p>Click here to get some of my favorite quotes:</p>
+        <button onclick="addRandomQuote()">Give me a quote</button>
+        <div id="quote-container"></div>
```

Comment:
**_Totally optional if you have time and interest_**: since some of the quotes are quite long and others are short, when you click the button multiple times (when you're scrolled all the way to the bottom) the page jumps up and down to compensate for the changing quote length. Can you come up with a way to avoid this and provide a smoother user experience?